template_feed/Microsoft.DotNet.Test.ProjectTemplates.9.0/content/Playwright-MSTest-CSharp/.template.config/template.json

```diff
@@ -72,6 +72,20 @@
   ],
   "defaultName": "TestProject1",
   "postActions": [
+    {
+      "description": "Add MSTest.Sdk entry to global.json",
+      "manualInstructions": [ { "text": "Add a new property 'MSTest.Sdk' under 'msbuild-sdks' section of the 'global.json' file." }],
+      "actionId": "695A3659-EB40-4FF5-A6A6-C9C4E629FCB0",
+      "args": {
+        "jsonFileName": ".\\global.json",
```

Comment:
I wonder if this syntax works on Linux or if the forward slash will be treated as part of the filename?